setup.py

```diff
@@ -33,6 +33,7 @@ def package_data(pkg, roots):
     ],
     install_requires=[
         'XBlock',
+        'rocketchat-API==0.6.34',
```

Comment:
This probably doesn't work with the rocketchat version installed on the servers. Could you test all the requests that the xblock makes ?

Reply:
It does not matter, we must upgrade the version of the xblock and install it in the Pearson instances, the current installations will work with the previous version.

Reply:
if you can not ensure that it will works with the previous version, please use the version 0.6.21. for the following reasons
1) you have not test all the api calls so there is a possibility that some methods need to be changed.
2) There's no ticket to update the pearson rocketchat server, so probably they wants to use the same one.
3) Maybe in the future they will want to change or add something to rocketchat in the shared instance.

Reply:
First, I've tested the xblock in the Ironwood instance and it works, shared platform is Ginkgo, we are upgrading to Ironwood, if they need anything for the old version instance (Ginkgo) we must create another branch for that and making the changes there and move forward with master.
We can't handle the same xblock version for two different platform version you know that. The version that works for Ginkgo is the previous one, the version that will work for Ironwood is this one.
Every time we can upgrade to a patch version we must update since those changes includes patch updates.

Reply:
Did you test create channels in the teams tab?

Reply:
Tab? This is the xblock.

Reply:
There is a feature that includes the xblock in the teams page, but you didn't know that, so you couldn't test it.So please ensure that the way, how the api calls are made for the xblock, has not changed, if you don't want, use the version 0.6.21

Reply:
Checked.